metadata/metadata_service/proxy/atlas_proxy.py

```diff
@@ -860,6 +902,12 @@ def _get_resources_followed_by_user(self, user_id: str, resource_type: str) \
         :param resource_type: Type of a resource that returns, could be table, dashboard etc.
         :return: A list of PopularTable, DashboardSummary or any other resource.
         """
+
+        if resource_type == ResourceType.Table.name:
+            bookmark_qn_search_pattern = f'_{resource_type.lower()}.{user_id}.bookmark'
+        else:
+            bookmark_qn_search_pattern = f'/{resource_type.lower()}/{user_id}/bookmark'
```

Comment:
as above, can we change user_id and bookmark positions?

Reply:
changed